iDiff/differs/differs.go

```diff
@@ -31,18 +34,41 @@ var diffs = map[string]Differ{
 	"node":    NodeDiffer{},
 }
 
-func (diff DiffRequest) GetDiff() (DiffResult, error) {
+func (diff DiffRequest) GetDiff() (map[string]DiffResult, error) {
 	img1 := diff.Image1
 	img2 := diff.Image2
-	differ := diff.DiffType
-	return differ.Diff(img1, img2)
-}
+	diffs := diff.DiffTypes
+
+	results := map[string]DiffResult{}
+	for _, differ := range diffs {
+		differName := reflect.TypeOf(differ).Name()
+		if diff, err := differ.Diff(img1, img2); err == nil {
+			results[differName] = diff
+		} else {
+			glog.Errorf("Error getting diff with %s", differName)
```

Comment:
Can you log the error from `differ.Diff()` here? That way if a diff fails we actually know what went wrong.